api/campaigns/class-maybe-show-campaign.php

```diff
@@ -325,10 +325,10 @@ public function should_popup_be_shown( $client_id, $popup, $settings, $referer_u
 			$frequency_reset   = 'month';
 		}
 		if ( 'preset_1' === $frequency ) {
```

Comment:
Since the `preset_1` is now removed in the options, there should be a comment indicating that this is a legacy setting.